### PR TITLE
[BugFix](table-value-function) Fix backends() tvf

### PR DESCRIPTION
**Pull request by BePPPower**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Change the `Alive/SystemDecommissioned/ClusterDecommissioned` field type of the `backends()`tvf to bool

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

File: regression-test/suites/correctness_p0/table_valued_function/test_backends_tvf.groovy [hunk 1]
```diff
@@ -25,25 +25,25 @@ suite("test_backends_tvf") {
...
     assertTrue(table[0].size == 4) // column should be 26
-    assertEquals("true", table[0][2])
+    assertEquals(true, table[0][2])
 
     // test changing position of columns
     table = sql """ select Hostname as name, NodeRole as r, alive, ip from backends();"""
     assertTrue(table.size() > 0) // row should > 0
     assertTrue(table[0].size == 4) // column should be 26
-    assertEquals("true", table[0][2])
+    assertEquals(true, table[0][2])
```
Comment (morningman) [inline regression-test/suites/correctness_p0/table_valued_function/test_backends_tvf.groovy:46]: Please add count(*) case

Comment (BePPPower) [reply regression-test/suites/correctness_p0/table_valued_function/test_backends_tvf.groovy:46]: done

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Comment by BePPPower** [comment]
run buildall

**Pull request merged**